iquip/apps/builder.py

```diff
@@ -375,17 +404,120 @@ def value(self) -> Dict[str, Any]:
         selected = self.state["selected"]
         return self.state[selected]
 
+    @pyqtSlot(QAbstractButton)
+    def scanTypeClicked(self, selectedButton: QAbstractButton):
+        """Switches current scan widget at stacked layout in _ScanEntry.
+        
+        Once the checked button at radiobutton group clicked, this is called.
```

Comment:
It looks a little weird.
How about "Once a scan type button is clicked, ..."?

Reply:
If you want to use "at radiobutton group", "in the button group" looks better.
And, please write "is clicked".

Reply:
@Aijuh